src/api/spec/cassettes/Workflow/_steps/with_one_unsupported_step/1_2_5_2.yml

```diff
@@ -816,12 +816,12 @@ http_interactions:
     body:
       encoding: UTF-8
       string: |
-        <sourcediff key="3b01b2f7b34d95332d49cebaeb06bcb3">
+        <sourcediff key="cc178dcc4febcacc4dd2e91769464db8">
           <old project="test-project" package="test-package" rev="d41d8cd98f00b204e9800998ecf8427e" srcmd5="d41d8cd98f00b204e9800998ecf8427e"/>
           <new project="test-project:iggy:test-project:PR-1" package="test-package" rev="3f2794130d6c60ddbfc7b7caa9ff69c6" srcmd5="3f2794130d6c60ddbfc7b7caa9ff69c6"/>
           <files/>
           <issues>
           </issues>
         </sourcediff>
-  recorded_at: Wed, 02 Aug 2023 10:05:55 GMT
-recorded_with: VCR 6.2.0
+  recorded_at: Thu, 13 Feb 2025 15:54:09 GMT
+recorded_with: VCR 6.3.1
```

Comment:
There are no changes in this cassette. The previous version should work.

Reply:
This is a new cassette now. Previously it was renamed with an existing cassette